data/mods/Aftershock/items/weapons.json

```diff
@@ -416,5 +416,23 @@
       [ "dampening", 1 ]
     ],
     "pocket_data": [ { "pocket_type": "MAGAZINE", "ammo_restriction": { "bolt": 1 } } ]
+  },
+  {
+    "id": "afs_bio_whip_weapon",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "red",
+    "name": { "str": "monofilament whip" },
+    "description": "A long filament of nested carbon nanotubes only a few microns in diameter.  In essence a very thin, very strong and very sharp blade that you throw at a moment's notice.",
+    "weight": "40 g",
+    "volume": "200 ml",
+    "price": 3500,
+    "price_postapoc": 500,
+    "material": [ "nanotubes" ],
+    "flags": [ "REACH_ATTACK", "REACH3", "NONCONDUCTIVE", "BIONIC_WEAPON", "UNBREAKABLE_MELEE", "TRADER_AVOID" ],
+    "techniques": [ "WBLOCK_1", "WIDE", "SPIN", "RAPID" ],
+    "cutting": 27,
+    "to_hit": { "grip": "bad", "length": "long", "surface": "every", "balance": "clumsy" },
```

Comment:
I think you nailed the to_hit on the hit.  It's awkward but dangerous.

Reply:
Yeah I think the techniques could be gated under efficiencies once theres support for that even.